data/tlg0527/tlg048/__cts__.xml

```diff
@@ -1,7 +1,13 @@
-<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0527" xml:lang="grc" urn="urn:cts:greekLit:tlg0527.tlg048">
-  <ti:title xml:lang="eng">Isaias</ti:title>
-  <ti:edition urn="urn:cts:greekLit:tlg0527.tlg048.opp-grc2" workUrn="urn:cts:greekLit:tlg0527.tlg048">
-    <ti:label xml:lang="eng">Isaias</ti:label>
-    <ti:description xml:lang="eng">Septuaginta, Isaias</ti:description>
+<ti:work xmlns:cpt="http://capitains.github.io/xmlns" xmlns:dct="http://purl.org/dc/terms/" xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0527" xml:lang="grc" urn="urn:cts:greekLit:tlg0527.tlg048">
+  <ti:title xml:lang="lat">Isaias</ti:title>
+  <cpt:structured-metadata>
+    <dct:isVersionOf>urn:cts:hebrewLit:heb0001.heb010</dct:isVersionOf>
+  </cpt:structured-metadata>
+  <ti:edition  urn="urn:cts:greekLit:tlg0527.tlg048.opp-grc2" workUrn="urn:cts:greekLit:tlg0527.tlg048">
+    <ti:label xml:lang="lat">Isaias</ti:label>
+    <ti:description xml:lang="mul">Septuaginta in English, Isaias, Ottley, Cambridge, 1904</ti:description>
```

Comment:
Delete the "in English" part from the edition's description. I think just having "Septuaginta" here is better.